packages/client/src/store/index.ts

```diff
@@ -1,21 +1,62 @@
-import { proxy, subscribe } from 'valtio';
-import type { Note } from '../types';
+import type { Line, Note } from '../types';
 import { removeNullish } from '../utils';
-import { getNewestLocalState, NoteTimeline } from './timelines/note';
+import { sendLinesUpdate, sendNotesUpdate } from './hooks';
+import { getEffectiveLine, LineTimeline } from './timelines/line';
+import { getEffectiveNote, NoteTimeline } from './timelines/note';
 
-type Store = {
-  noteTimelines: { [noteId: string]: NoteTimeline };
+const noteTimelines: { [noteId: string]: NoteTimeline } = Object.create(null);
+const lineTimelines: { [lineId: string]: LineTimeline } = Object.create(null);
+
+export const updateNotes = <T>(
+  callback: (nTimelines: typeof noteTimelines) => T
+): T => {
+  // Legacy function from before valtio introduction
```

Comment:
I like that this is considered "legacy" already :smile:

Reply:
Haha. Thanks for pointing out. This comment is legacy and should be removed :joy:  Because I didin't use valtio in the end

Reply:
Rabbit hole goes deep I see :sweat_smile: